src/Assert.php

```diff
@@ -813,6 +832,42 @@ public static function count($array, $number, $message = '')
         );
     }
 
+    public static function minCount($array, $min, $message = '')
+    {
+        if (count($array) < $min) {
```

Comment:
Maybe a check if it is countable before?

`static::isCountable($array, $message)`